cmd/podman/runlabel.go

```diff
@@ -23,7 +25,7 @@ var (
 		},
 		cli.BoolFlag{
 			Name:  "display",
-			Usage: "preview the command that `podman install` would execute",
+			Usage: "preview the command the label's command",
```

Comment:
"preview the command the label would run" maybe?

Reply:
ok